Auxiliary/DirectXTexEXR.cpp

```diff
@@ -265,14 +265,25 @@ HRESULT DirectX::GetMetadataFromEXRFile(const wchar_t* szFile, TexMetadata& meta
         const auto dw = file.dataWindow();
 
         const int width = dw.max.x - dw.min.x + 1;
-        const int height = dw.max.y - dw.min.y + 1;
+        int height = dw.max.y - dw.min.y + 1;
```

Comment:
A little bit of inconsistency between constness

Reply:
Width can be const. Height cannot.